vars/main.yml

```diff
@@ -8,3 +8,7 @@ postgresql_user: thoth
 postgresql_database: postgres
 postgresql_volume_capacity: "1Gi"
 postgresql_version: "9.6"
+storage_class_name: ceph-dyn-thoth-paas-prod-psi
+ceph_pv_secret_name: ceph-thoth-paas-prod-psi-admin-secret
+# placeholder
+ceph_pv_secret: ceph-dyn-thoth-paas-prod-psi
```

Comment:
should we remove this default to not accidentally configure some insecure value and force the user to provide a value?

Reply:
yes, that make sense :+1: